migrations/mystery_items.js

```diff
@@ -27,6 +27,7 @@ if (_id) {
   db.users.update({_id:_id}, update);
 } else {
   // multiple (once @ start of event)
-  db.users.update({'purchased.plan.customerId':{$ne:null}}, update, {multi:true});
-}
-
+  db.users.find({'purchased.plan.customerId':{$ne:null}}).forEach(function(user) {
```

Comment:
Looking at the docs again, I don't think it should be necessary, and I can't recall why I added it in the first place. I'll make the change shortly - I think I will just move the customerId check back into the update query in an $and clause.